tests/bootstrap.php

```diff
@@ -12,8 +10,6 @@
 
 require_once __DIR__ . '/../vendor/autoload.php';
 
-ClassDiscovery::appendStrategy(MockClientStrategy::class);
```

Comment:
"well-known-implem" will discover and auto-load "php-http/mock-client" when no other implem is found.
This decouples from "php-http/discovery".